dotcom-rendering/src/web/components/ArticleMeta.tsx

```diff
@@ -38,28 +38,48 @@ type Props = {
 	showShareCount: boolean;
 };
 
-const meta = css`
-	${between.tablet.and.leftCol} {
-		order: 3;
-	}
+const meta = (format: ArticleFormat) => {
+	if (
+		format.design === ArticleDesign.LiveBlog ||
+		format.design === ArticleDesign.DeadBlog
+	) {
+		return css`
+			${between.tablet.and.leftCol} {
+				order: 3;
+			}
 
-	${until.mobileLandscape} {
-		padding-left: 10px;
-		padding-right: 10px;
-	}
+			${from.phablet} {
+				padding-left: 0px;
+				padding-right: 0px;
+			}
```

Comment:
Shouldn’t this be simply `padding-left: 0px;` without the media query? Or simply drop this directive entirely?

Reply:
Thanks! Done